vagrant/pythonstartup.py

```diff
@@ -19,5 +19,7 @@ def save_history():
 try:
     import sideboard
     from uber.common import *
+    session = Session().session  # make it easier to do session stuff at the command line
```

Comment:
This next line lets us say stuff like

```python
>>> from uber.site_sections import jobs
>>> jobs.Root().index()  # manually generate the index page here
```

This makes it much easier to test things sometimes.